src/RuleSet.php

```diff
@@ -203,7 +210,10 @@
      */
     private $rules;
 
-    public function __construct(array $set = [])
+    /**
+     * {@inheritdoc}
+     */
+    public function __construct(array $set = [], array $customSetDefinitions = [])
```

Comment:
it could be nice moment do decouple definitions from ruleset that is suppose to be used during fixing